motion.go

```diff
@@ -21,6 +21,8 @@ type motionDetector struct {
 	deltaThresh  uint16
 	countThresh  int
 	nonzeroLimit int
+	tarX         int
+	tarY         int
```

Comment:
what does `tar` mean? perhaps use a clearer name?

Reply:
Target x and y position in current frame. I will change them to `targetX` and `targetY`

Reply:
yes please